client/components/app-promo/test/index.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
```

Comment:
By convention, imports should be decorated with `External dependencies` and `Internal dependencies` blocks.